Env command usage

help says:
```
USAGE
  $ ssmenv env STAGE
```

but I get:
```
$ ssmenv env dev
Error: Cannot find module 'tslib'
    at Object.<anonymous> (~/.nvm/versions/node/v9.9.0/lib/node_modules/ssmenv/lib/commands/env/index.js:3:17)
```

if I use: 
```
$ ssmenv env:dotenv dev
```

I get: 
```
THING2=blue 0 [ 'THING2=blue', 'THING1=5', 'BLARG=true' ]
THING1=5 1 [ 'THING2=blue', 'THING1=5', 'BLARG=true' ]
BLARG=true 2 [ 'THING2=blue', 'THING1=5', 'BLARG=true' ]
```

--- a/src/commands/env/index.ts
+++ b/src/commands/env/index.ts
@@ -1,1 +1,3 @@
-export * from './dotenv';
+import { EnvDotenv } from './dotenv';
+
+export class EnvIndex extends EnvDotenv {}

--- a/src/commands/var/index.ts
+++ b/src/commands/var/index.ts
@@ -1,1 +1,3 @@
-export * from './set';
+import { VarSet } from './set';
+
+export class VarIndex extends VarSet {}

